python/katana/benchmarking/bench_python_cpp_algos.py

```diff
@@ -469,9 +468,8 @@ def main(parsed_args):
 
 
 if __name__ == "__main__":
-    options = initialize_global_vars()
-    all_apps = options[0]
-    all_graphs = options[1]
+    all_apps, all_graphs = initialize_global_vars()
```

Comment:
Maybe move the entire body of this block into a function (for example called `main`), so we can call it nicely from tests; we do  not want to start the script.